src/backend/package.json

```diff
@@ -21,9 +21,11 @@
     "test:e2e": "jest --config ./test/jest-e2e.json"
   },
   "dependencies": {
+    "@nestjs/axios": "0.1.0",
     "@nestjs/common": "9.0.0",
     "@nestjs/core": "9.0.0",
     "@nestjs/platform-express": "9.0.0",
+    "@nestjs/terminus": "9.1.2",
```

Comment:
Is it necessary to have terminus? Can we keep the default nestjs package.json without adding more to this ginormous list?

Reply:
it depends on what health check we want to do in the future, NestJS provide many different indicators to monitor our system, like database, CPU, mem, etc even we can define our custom health check indicator. so yes, if only a simple http health check, we do not need terminus, but if we do want to implement a sophisticated HC, I think we need this tool.

Reply:
I think we should delegate all the health monitoring related heavy lifting to Openshift.

There can be multiple node instances in future - we do not want all the nodes to take care of these types of monitoring using resources. What do you think?